### ISSUE TITLE
Code2flow installs improperly, is added to PATH, and opens "How do you want to open this file?" menu

### ISSUE THREAD

**Issue by Shidouuu** [scottrogowski/code2flow#24]
Following the instructions in the README, using `python install setup.py`, code2flow is added to PATH, but when trying to run, whether with a file or by itself, it instead opens a "How do you want to open this file?" menu. 

If you select a text editor it will bring up this:
```python
#!"C:\Users\[User]\AppData\Local\Programs\Python\Python39\python.exe"
# EASY-INSTALL-SCRIPT: 'code2flow==2.0.1','code2flow'
__requires__ = 'code2flow==2.0.1'
__import__('pkg_resources').run_script('code2flow==2.0.1', 'code2flow')
```

Uninstalling and reinstalling it does nothing.

Using Python 3.9.1 on Windows 10.

**Comment by scottrogowski** [comment]
Hi @Shidouuu.

I cannot help debug this, unfortunately, because I don't have a windows computer. My feeling is that either your Python environment is setup incorrectly or that how you installed code2flow is not the regular "windows way" to do so. For what it's worth, the command should be `python setup.py install`, not `python install setup.py`.

**Comment by Shidouuu** [comment]
Log of `python setup.py install`:
```
running install
running bdist_egg
running egg_info
writing code2flow.egg-info\PKG-INFO
writing dependency_links to code2flow.egg-info\dependency_links.txt
writing top-level names to code2flow.egg-info\top_level.txt
reading manifest file 'code2flow.egg-info\SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'LICENSE,'
writing manifest file 'code2flow.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\lib
copying build\lib\lib\engine.py -> build\bdist.win-amd64\egg\lib
copying build\lib\lib\model.py -> build\bdist.win-amd64\egg\lib
copying build\lib\lib\python.py -> build\bdist.win-amd64\egg\lib
copying build\lib\lib\__init__.py -> build\bdist.win-amd64\egg\lib
byte-compiling build\bdist.win-amd64\egg\lib\engine.py to engine.cpython-39.pyc
byte-compiling build\bdist.win-amd64\egg\lib\model.py to model.cpython-39.pyc
byte-compiling build\bdist.win-amd64\egg\lib\python.py to python.cpython-39.pyc
byte-compiling build\bdist.win-amd64\egg\lib\__init__.py to __init__.cpython-39.pyc
creating build\bdist.win-amd64\egg\EGG-INFO
installing scripts to build\bdist.win-amd64\egg\EGG-INFO\scripts
running install_scripts
running build_scripts
creating build\bdist.win-amd64\egg\EGG-INFO\scripts
copying build\scripts-3.9\code2flow -> build\bdist.win-amd64\egg\EGG-INFO\scripts
copying code2flow.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying code2flow.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying code2flow.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying code2flow.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist\code2flow-2.0.1-py3.9.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing code2flow-2.0.1-py3.9.egg
Copying code2flow-2.0.1-py3.9.egg to c:\users\guitar god\appdata\local\programs\python\python39\lib\site-packages
Adding code2flow 2.0.1 to easy-install.pth file
Installing code2flow script to C:\Users\Guitar God\AppData\Local\Programs\Python\Python39\Scripts

Installed c:\users\guitar god\appdata\local\programs\python\python39\lib\site-packages\code2flow-2.0.1-py3.9.egg
Processing dependencies for code2flow==2.0.1
Finished processing dependencies for code2flow==2.0.1
```

**Comment by scottrogowski** [comment]
I've got a possible fix that seems to be "the way" to make CLI tools run in a cross-platform way.

Could you do me a favor, download the windows branch, and let me know if that fixes the problem? https://github.com/scottrogowski/code2flow/tree/windows

**Comment by Shidouuu** [comment]
It doesn't open the "How do you want to..." dialog anymore but unfortunately it still only outputs the legend. Might be a different issue, I'm not sure.

**Comment by scottrogowski** [comment]
Ah, I missed the "it only outputs the legend" part. So it's working. Looking at your log output, it looks like it only found the implicit (global) node which means you don't have any defined functions in your file or, alternately, the defined functions are nested in a complicated way. Code2flow doesn't connect to functions in any file that wasn't input - including third-party modules. It looks like the only calls you have are to one or more third-party packages (is it PIL?). 

Closing the issue for now as it seems the main issue has been resolved. Feel free to open another if I got that wrong.
